scripts/deploy_gcf.py

```diff
@@ -80,6 +80,17 @@ class GoogleRuntimeConfigConnection(JSONConnection):
                                       "Condition": {"ArnLike": {"aws:SourceArn": sender_arn}}}]}
 sync_sqs_queue.set_attributes(Attributes=dict(Policy=json.dumps(queue_access_policy)))
 
+index_sqs_queue_name = "dss-index-" + os.environ["DSS_DEPLOYMENT_STAGE"]
+index_sqs_queue = boto3_session.resource("sqs").create_queue(QueueName=index_sqs_queue_name)
+sender_arn = f"arn:aws:sns:*:{aws_account_id}:{relay_sns_topic_name}"
+queue_access_policy = {"Statement": [{"Sid": "dss-deploy-gcf-qap",
```

Comment:
Yes, these IAM policies could moved to iam/policytemplate. However, that would further fragment the deployment of the google cloud function.

It's also possible to place IAM policies in the daemon/chalice directories, e.g. `/daemons/dss-gs-event-relay/gcf_iam_policies.tf`. This would create a much more modular deployment system. Something to think about.